client/src/components/SearchPage/SearchField.js

```diff
@@ -0,0 +1,111 @@
+import React, {Component} from 'react';
+import './SearchField.css'
+import Logo from '../assets/logo.png'
+import Search from '../assets/search.png';
+import Mic from '../assets/mic.png'
+import Suggestion from "./data";
+import data from './data'
+class SearchField extends Component{
+
+
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            words: [],
+            userInput: '',
+            matches: [],
+            rowHighlighted: -1
+        }
+    }
+
+    componentDidMount(){
+        this.setState({
+            words: data.words
+        })
+    }
+
+    handleUserInput(e) {
+        let input = e.target.value;
```

Comment:
You don't seem to be changing the value of `input` down there. Why use `let` then? Use `const`. If you're not gonna be changing a variable later, declare it with `const`.
Again, this *also* improves code readability.